features/bootstrap/FeatureContext.php

```diff
@@ -268,8 +268,8 @@ public function iRunBehatInDebugMode()
      */
     public function itShouldPassWith($success, PyStringNode $text)
     {
-        $this->itShouldFail($success);
         $this->theOutputShouldContain($text);
+        $this->itShouldFail($success);
```

Comment:
I changed the order of this and the check below since comparing the output first is more informative and tends to be more helpful when debugging tests.

Reply:
@uuf6429 this part looks reasonable, if you want to submit as a separate PR?

For future contributions, it would help if you could group unrelated changes into separate commits, with the explanation in the commit message rather than a github comment. That way we don't lose that information if/when the PR is merged. Thanks :)

Reply:
> For future contributions, it would help if you could group unrelated changes into separate commits

It's unfortunate that these sort of things pop up during development of unrelated tasks. Normally when the change is too big or controversial, I would open a separate issue. When it's something really minor (and directly covered by tests), I'd just go for it since it doesn't seem to be worth the effort to have a different PR.

But anyway, it's fine, let's take that approach. I'll extract the uncontroversial changes into separate PRs.

Reply:
Just to be clear, I'm fine with small things being in the same PR so long as separate concerns are in separate commits (with their own commit messages). That way even if we disagree on what's small/groupable then worst case is we split the commits to separate branches during review, which is easy, rather than having to ask people to go back over things.

I'd only suggested putting this into a separate PR to get it merged even though I've declined the other changes. If it was a separate commit already I'd probably just have cherry picked it myself.

Reply:
> Just to be clear, I'm fine with small things being in the same PR so long as separate concerns are in separate commits (with their own commit messages).

Ah, I didn't get that. All good then.

I've extracted the code, by the way. Thanks.